smarts/core/agent_manager.py

```diff
@@ -41,9 +41,11 @@ class AgentManager:
          time.
     """
 
-    def __init__(self, interfaces, zoo_workers=None):
+    def __init__(self, interfaces, zoo_workers=None, authkey=b"secret"):
```

Comment:
```suggestion
    def __init__(self, interfaces, zoo_workers=None, auth_key=None):
```

Reply:
Changed default value of `auth_key` of type byte string to be `b"None"`.